commit 35c01ca9176d9fae823b2b6cfd4f6d49717dbe99
Author: Robert Mustacchi <rm@joyent.com>
Date:   2019-05-07T18:25:33+00:00 (5 months ago)
    
    OS-7669 Want CPU Temperature Sensors
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/overlay/generic/etc/driver_aliases b/overlay/generic/etc/driver_aliases
index 62e8bda2..0ac01ecc 100644
--- a/overlay/generic/etc/driver_aliases
+++ b/overlay/generic/etc/driver_aliases
@@ -67,6 +67,8 @@ amd64_gart "pci1022,1103"
 amd8111s "pci1022,7462"
 amd_iommu "pci1002,5a23"
 amd_iommu "pci1022,11ff"
+amdf17nbdf "pci1022,1450"
+amdf17nbdf "pci1022,1460"
 amr "pci1000,1960.1000,532"
 amr "pci101e,1960.1028,493"
 amr "pci1000,1960.1028,518"
@@ -1422,6 +1424,6 @@ yge "pciex11ab,4355"
 yge "pciex11ab,4362"
 yge "pciex11ab,4363"
 yge "pciex11ab,4364"
-yge "pciex11ab,436a"	
+yge "pciex11ab,436a"
 vioblk "pci1af4,1001"
 vioif "pci1af4,1"
diff --git a/overlay/generic/etc/name_to_major b/overlay/generic/etc/name_to_major
index 26eaeddb..a92bd1e2 100644
--- a/overlay/generic/etc/name_to_major
+++ b/overlay/generic/etc/name_to_major
@@ -302,3 +302,5 @@ buildstamp 304
 vmm 305
 viona 306
 ppt 307
+coretemp 308
+amdf17nbdf 309
